awx/plugins/isolated/awx_isolated_cleanup.py

```diff
@@ -51,9 +51,10 @@ def main():
                 try:
                     re_match = re.match(r'\/tmp\/awx_\d+_.+', path)
                     if re_match is not None:
-                        if subprocess.check_call(['ansible-runner', 'is-alive', path]) == 0:
-                            continue
```

Comment:
Do we still need the `continue` with your change?

Reply:
Yes 🤦‍♂